cop.h

```diff
@@ -228,6 +229,58 @@ string/length pair.
 #define cophh_fetch_sv(cophh, key, hash, flags) \
     Perl_refcounted_he_fetch_sv(aTHX_ cophh, key, hash, flags)
 
+/*
+=for apidoc Amx|SV *|cophh_exists_pvn|const COPHH *cophh|const char *keypv|STRLEN keylen|U32 hash|U32 flags
```

Comment:
Shouldn’t this say `bool` rather than `SV *` for its return type? @Leont

Reply:
Yeah, updated that